plugin/src/main/java/org/opennms/velocloud/ApiClient.java

```diff
@@ -1,13 +1,46 @@
 package org.opennms.velocloud;
 
+import java.util.Map;
 import java.util.concurrent.CompletableFuture;
 
-import org.opennms.velocloud.model.Alert;
+import org.opennms.velocloud.client.handler.Configuration;
+import org.opennms.velocloud.client.handler.auth.ApiKeyAuth;
+import org.opennms.velocloud.client.handler.auth.HttpBasicAuth;
+import org.opennms.velocloud.client.handler.auth.OAuth;
 import org.opennms.velocloud.model.Topology;
 
-public class ApiClient {
+public class ApiClient extends org.opennms.velocloud.client.handler.ApiClient {
+
+    public static final String AUTHORIZATION = "Authorization";
+    public static final String HEADER = "header";
+    public static final String TOKEN = "Token";
+
+    public ApiClient(final String url, final String params, final String authType) {
```

Comment:
Do not over-complicate this. Currently we just know API keys to work. Let's stick with this as the single option and reflect it in the constructors signature.